pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -136,12 +134,21 @@ func (lp *ProxyLifecycleProcessor) spanModifier(lastReqId string, chunk *pb.Trac
 	}
 	log.Debugf("appsec: found service entry span of the currently monitored request id `%s`", currentReqId)
 
+	span := (*spanWrapper)(s)
```

Comment:
Wouldn't it be better practice to declare this in the `lp.invocationEvent == nil` branch, since it's only used there?

Reply:
It's used in both "branches" so I moved it upper now that I need it in the unsupported event condition below too now.